src/sql/src/plan/statement/ddl.rs

```diff
@@ -51,8 +59,12 @@ use crate::catalog::{CatalogItem, CatalogItemType};
 use crate::kafka_util;
 use crate::names::{DatabaseSpecifier, FullName, SchemaName};
 use crate::normalize;
+use crate::plan::expr::ColumnRef;
+use crate::plan::expr::HirScalarExpr;
+use crate::plan::expr::JoinKind;
 use crate::plan::query::QueryLifetime;
 use crate::plan::statement::{StatementContext, StatementDesc};
+use crate::plan::HirRelationExpr;
```

Comment:
Can you configure your Rust Analyzer to merge imports with the "last" configuration setting? I know you don't care over in `dataflow` about import sort order, but we do in `sql`, and I think just setting that setting might keep everyone happy.

Reply:
sure